keanu-project/src/main/java/io/improbable/keanu/templating/SequenceItem.java

```diff
@@ -20,9 +20,19 @@
     private static Pattern NAME_REGEX = Pattern.compile(NAME_PREFIX + "-?[\\d]+$");
 
     private Map<VertexLabel, Vertex<?>> contents;
+    private int itemPosition;
+    private int uniqueIdentifier;
+    private String identifyingNamespace;
```

Comment:
I'd probably just call this `sequenceName` - the fact that it's used as the namespace for a VertexLabel is an implementation detail.